### ISSUE TITLE
Add default placement option for webbapp AND open webbapp windows in latest format.

### ISSUE THREAD

**Issue by gotolindak73** [linuxmint/webapp-manager#165]
I quite like the Webbapp feature in Linux Mint. I have noticed since two annoying issues though. I run Linux Mint Cinnamon 20.3 and when I open my webbapp it always opens in the left half of my screen and always in the same format. Is it possible to get the window to open center AND remember the latest windowformat? I have tried setting the Windows placement to Strikt Center and that works fine with regular apps but not with webbapps.

**Comment by mchilli** [comment]
hey maybe its the same issue as here: #163

**Comment by gotolindak73** [comment]
Yes it seems to be just that. So Chrome is the issue. Tested to add a new webbappa with firefox and I can se that it works fine with Firefox. Thank you for responding. :-)
